.git is pusblished at npm

The latest build under gitlab CI throw me the following error:

```bash
$ NODE_ENV=development npm install
npm ERR! path /node_modules/@ffprobe-installer/ffprobe
npm ERR! code EISGIT
npm ERR! git /node_modules/@ffprobe-installer/ffprobe: Appears to be a git repo or submodule.
npm ERR! git     /node_modules/@ffprobe-installer/ffprobe
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-05-30T17_02_51_899Z-debug.log
```

The error is saying npm doesn't like an npm package with `.git` folder inside.

I did a fresh install to check if the `.git` folder has been part of the npm package, and there it is:

```bash
~/Downloads/test/node_modules/@ffprobe-installer/ffprobe
❯ l
drwxr-xr-x    - josefranciscoverdugambin staff 30 May 19:28 ./
.rw-r--r-- 6.1k josefranciscoverdugambin staff 26 Oct  1985 ├── .DS_Store
.rw-r--r--  205 josefranciscoverdugambin staff 26 Oct  1985 ├── .editorconfig
drwxr-xr-x    - josefranciscoverdugambin staff 30 May 19:28 ├── .git/
.rw-r--r--  116 josefranciscoverdugambin staff 26 Oct  1985 ├── .gitattributes
drwxr-xr-x    - josefranciscoverdugambin staff 30 May 19:28 ├── .nyc_output/
.rw-r--r--   78 josefranciscoverdugambin staff 26 Oct  1985 ├── .tidelift.yml
.rw-r--r--  168 josefranciscoverdugambin staff 26 Oct  1985 ├── .travis.yml
.rw-r--r--  429 josefranciscoverdugambin staff 26 Oct  1985 ├── appveyor.yml
.rw-r--r--  660 josefranciscoverdugambin staff 26 Oct  1985 ├── coverage.lcov
.rw-r--r-- 1.3k josefranciscoverdugambin staff 26 Oct  1985 ├── index.js
drwxr-xr-x    - josefranciscoverdugambin staff 30 May 19:28 ├── lib/
.rw-r--r-- 2.4k josefranciscoverdugambin staff 30 May 19:28 ├── package.json
.rw-r--r-- 4.1k josefranciscoverdugambin staff 26 Oct  1985 ├── README.md
.rw-r--r--  465 josefranciscoverdugambin staff 26 Oct  1985 └── test.js
```

Ideally, we can declare `files` field at `package.json` to just be sure a subset of files are shipped at npm:

```
{
  "files": [
    "index.js",
    "lib",
    "platform"
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -14,6 +14,11 @@
 	"author": "Oliver Sayers <talk@savagecore.eu>",
 	"license": "LGPL-2.1",
 	"description": "Platform independent binary installer of FFprobe for node projects",
+	"files": [
+    "index.js",
+    "lib",
+    "platform"
+  ],
 	"optionalDependencies": {
 		"@ffprobe-installer/darwin-x64": "4.1.0",
 		"@ffprobe-installer/linux-ia32": "4.1.0",

--- a/package.json
+++ b/package.json
@@ -15,10 +15,10 @@
 	"license": "LGPL-2.1",
 	"description": "Platform independent binary installer of FFprobe for node projects",
 	"files": [
-    "index.js",
-    "lib",
-    "platform"
-  ],
+		"index.js",
+		"lib",
+		"platform"
+	],
 	"optionalDependencies": {
 		"@ffprobe-installer/darwin-x64": "4.1.0",
 		"@ffprobe-installer/linux-ia32": "4.1.0",

